### PR TITLE
Modified script to combine root and seccheck env and run it from NFS

### PR DESCRIPTION
**Pull request by kumarnarendra701**
- What I did
    Combine root and secchek env.

- How I did it
    Combine all env to /root/env and added reference in seccheck script.

- How to verify it
    Cronjob is running from the root user

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-<img width=250px src="https://atsign.dev/assets/img/atPlatform_logo_gray.svg?sanitize=true">
+<img src="https://atsign.dev/assets/img/@dev.png?sanitize=true">
```
Comment (cpswan) [inline CONTRIBUTING.md]: This file shouldn't be changed, and the proposed change is reverting to an old version.

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,6 @@
 BSD 3-Clause License
 
-Copyright (c) 2022, The Atsign Foundation
+Copyright (c) 2021, The @ Foundation
```
Comment (cpswan) [inline LICENSE]: Reversion

File: README.md [hunk 3]
```diff
@@ -1,4 +1,6 @@
-<img width=250px src="https://atsign.dev/assets/img/atPlatform_logo_gray.svg?sanitize=true">
+<img src="https://atsign.dev/assets/img/@dev.png?sanitize=true">
```
Comment (cpswan) [inline README.md]: Reversion

**Review by cpswan — CHANGES_REQUESTED** [review]
There are a bunch of files here that have been changed back to old versions. Please make sure you're working from latest trunk before creating a branch for changes.

I'll take a proper look at the intended changes once they're not surrounded by all the wrong stuff.

File: CONTRIBUTING.md (hunk 1 above)
Comment (kumarnarendra701) [reply CONTRIBUTING.md]: Fixed

File: LICENSE (hunk 2 above)
Comment (kumarnarendra701) [reply LICENSE]: Fixed

File: README.md (hunk 3 above)
Comment (kumarnarendra701) [reply README.md]: Fixed

**Review by cpswan — APPROVED** [review]

**Pull request merged**